applications/serial_modem/README.rst

```diff
@@ -4,7 +4,6 @@ Serial Modem
 ############
 
 The Serial Modem contains the Serial Modem application for nRF91-based devices, and is built on `nRF Connect SDK`_.
-It includes host examples in the form of the `Serial Modem Host`_ library and the `Serial Modem Host Shell`_ sample.
```

Comment:
Shouldn't we link  to the SM AT client sample and SM AT Client lib here and in the below cases?